rust/src/memory_pool.rs

```diff
@@ -0,0 +1,54 @@
+use libc;
```

Comment:
You need to add an Apache license header as we have in all other files.

Reply:
Fixed.